src/sentry/web/frontend/setup_wizard.py

```diff
@@ -23,13 +23,54 @@ def get(self, request, wizard_hash):
         This opens a page where with an active session fill stuff into the cache
         Redirects to organization whenever cache has been deleted
         """
-        context = {"hash": wizard_hash}
+        context = {"hash": wizard_hash, "mobile": False}
         key = f"{SETUP_WIZARD_CACHE_KEY}{wizard_hash}"
 
         wizard_data = default_cache.get(key)
         if wizard_data is None:
             return self.redirect_to_org(request)
 
+        if wizard_hash.startswith("mobile-"):
+            context = {"hash": wizard_hash, "mobile": True}
+            result = self.mobile_wizard(request, wizard_data)
+        else:
+            result = self.setup_wizard(request, wizard_data)
+
+        key = f"{SETUP_WIZARD_CACHE_KEY}{wizard_hash}"
+        default_cache.set(key, result, SETUP_WIZARD_CACHE_TIMEOUT)
+
+        return render_to_response("sentry/setup-wizard.html", context, request)
+
+    def mobile_wizard(self, request, wizard_data):
+        mobile_scope = [
+            "project:releases",
+            "project:read",
+            "event:read",
+            "team:read",
+            "org:read",
+            "member:read",
+            "alerts:read",
+        ]
```

Comment:
A nice by-product of this is that our in-app project bookmarking will become less hacky. 
We might want to add `project:write` here.